client/my-sites/email/email-providers-stacked-comparison/provider-cards/get-on-submit-new-mailboxes-handler.ts

```diff
@@ -25,6 +28,17 @@ export type GetOnSubmitNewMailboxesHandlerProps = {
 	source: string;
 };
 
+const getEmailProductPropertiesForUpsell = (
+	emailProduct: ProductListItem | null,
+	newMailboxesCount: number
+): EmailProperties => ( {
+	existingItemsCount: 0,
+	isAdditionalMailboxesPurchase: false,
+	emailProduct: emailProduct as ProductListItem,
+	newQuantity: newMailboxesCount,
+	quantity: 0,
```

Comment:
I'm not that familiar with our API, but should `quantity` really be `0` here?

Reply:
It shouldn't. 👍